.eslintrc.json

```diff
@@ -0,0 +1,7 @@
+{
+  "root": true,
+  "extends": [ "@tpluscode" ],
```

Comment:
What is `@tpluscode`? Is this a Zazuko-safe preset? :D

Reply:
Pretty much [standard+typescript](https://github.com/tpluscode/eslint-config/blob/master/index.js). Used to be `zazuko/esling-config` but politics... ;)